pywt/_thresholding.py

```diff
@@ -312,13 +319,17 @@ def _sigma_est_dwt(detail_coeffs, distribution='Gaussian'):
     # Consider regions with detail coefficients exactly zero to be masked out
     detail_coeffs = detail_coeffs[np.nonzero(detail_coeffs)]
 
-    if distribution.lower() == 'gaussian':
+    if hasattr(distribution, 'ppf'):
+        if not kwargs:
+            kwargs = {'q': 0.75}
```

Comment:
I originally thought it would be good to auto-fill the arguments with 75th quantile in the case there are no args passed in to keep up the defaults, but this may be a bad idea in case someone wants to use a ppf method which takes no arguments (as this would result in passing in an unwanted keyword argument).